vaadin-archetype-spring-application/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,40 @@
+# My App
+
+This project can be used as a starting point to create your own Vaadin application with Spring Boot.
+It contains all the necessary configuration and some placeholder files to get you started.
+
```

Comment:
Perhaps we can mention here, that for a full-configured project they can visit http://start.vaadin.com ?

Reply:
done